test/test_ss_parsimony_restraint.py

```diff
@@ -40,16 +40,25 @@ def setup_structural_element(root_hier, element, max_translation=1):
 
     #print(xyz)
 
-    IMP.threading.StructureElement.setup_particle(root_hier.get_model(), pi.get_index(), element[0]-4, 1, element[1], 0)
+#    IMP.threading.StructureElement.setup_particle(root_hier.get_model(), pi.get_index(), element[0]-4, 1, element[1], 0, 'H')
+    IMP.threading.StructureElement.setup_particle(root_hier.get_model(), pi.get_index(), element[0]-4, 1, element[1], 0, element[-1])
     #se = se.setup_particle(p, element[0], 1, element[1], 0)
 
     # Set up this element as a helix
-    IMP.atom.SecondaryStructureResidue.setup_particle(pi, 1, 0, 0)
+    if element[-1] == 'H':
+        IMP.atom.SecondaryStructureResidue.setup_particle(pi, 1, 0, 0)
+    elif element[-1] == 'S':
+        IMP.atom.SecondaryStructureResidue.setup_particle(pi, 0, 1, 0)
+    elif element[-1] == 'C':
+        IMP.atom.SecondaryStructureResidue.setup_particle(pi, 0, 0, 1)
```

Comment:
Could perhaps be written more succinctly as

    probmap = {'H': (1, 0, 0), 'S': (0, 1, 0), 'C': (0, 0, 1)}
    prob = probmap.get(element[-1], (0.33, 0.33, 0.33))
    IMP.atom.SecondaryStructureResidue.setup_particle(pi, *prob)